frontend/src/component/common/StaleDataNotification/StaleDataNotification.tsx

```diff
@@ -0,0 +1,75 @@
+import { Typography, Button, useTheme, useMediaQuery } from '@mui/material';
+import EventDiff from 'component/events/EventDiff/EventDiff';
+import { useThemeStyles } from 'themes/themeStyles';
+import AnimateOnMount from '../AnimateOnMount/AnimateOnMount';
+
+interface IStaleDataNotification {
+    refresh: () => void;
+    afterSubmitAction: Function;
```

Comment:
Would this be more consistent?
```suggestion
    afterSubmitAction: () => void;
```